app/models/supplejack_api/concerns/privacyable.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module SupplejackApi
+  module Concerns
+    module Privacyable
```

Comment:
keen to find a better name :)

Reply:
If it's about viewing restrictions it could be `AccessRestrictable` or just `Restrictable`? What you have is also fine. The `able` prefixing of concerns is debated, I use it and I am fine semi-broken English when it conveys the concern best (everyone would understand `Privacyable` the same way).

Reply:
Hmm, actually I think privacyable wins a good point having the word "privacy" which is the field. I'll leave it as it is.